packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -4260,7 +4260,7 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
   @override
   T? dependOnInheritedWidgetOfExactType<T extends InheritedWidget>({Object? aspect}) {
     assert(_debugCheckStateIsActiveForAncestorLookup());
-    final InheritedElement? ancestor = _inheritedWidgets == null ? null : _inheritedWidgets![T];
+    final InheritedElement? ancestor = _inheritedWidgets == null ? null : _inheritedWidgets!.lookup(T);
```

Comment:
Is there a reason that you called the operation `lookup` instead of `operator []`?

Reply:
I used `operator[]` originally but then switched to `lookup`. I don't have strong opinion either way.